SIGs.md

```diff
@@ -0,0 +1,33 @@
+# Special Interest Groups (SIGs)
+
+Semantic conventions in OpenTelemetry are organized into distinct areas,
+with each area being overseen by a Special Interest Group (SIG). A SIG is
+composed of multidisciplinary contributors who bring expertise in both the
+relevant domain and OpenTelemetry itself.
+
+SIGs have several responsibilities and goals, but their primary focus is to
+define and guide semantic conventions toward stabilization. They are also tasked
+with the long-term maintenance of these conventions, serving as domain experts
+for pull request (PR) reviews and issue triage.
+
+This document provides an overview of all SIGs involved in the
+semantic conventions project. In particular, the table below highlights
+the owners of each SIG and their current status.
+
+To learn more about SIGs for example, how to kick-start a new one, see the
```

Comment:
wonder if we can 
- add a warning that this table is under development
- merge this PR 
- divide and conquer filling SIG details

WDYT?